python/stalin_sort_functional.py

```diff
@@ -0,0 +1,36 @@
+from functools import reduce
+from typing import NamedTuple, List
+from math import inf
+
+
+class State(NamedTuple):
+    max_val: float = -inf
+    ordered_list: List[float] = []
+
+
+def sort_step(acc: State, number: float) -> State:
+    if acc.max_val < number:
```

Comment:
with repeated numbers, it should return them both 
![image](https://user-images.githubusercontent.com/732422/62377660-94fe1480-b519-11e9-9f10-a2113a50a8c0.png)
in this case, it should return `[1 2 3 3 6]`

Reply:
Hi :D. i didn't know that. i wiil fix.